### PR TITLE
Add locust script for scale testing (#542)

### PR DESCRIPTION
**Pull request by jessebrennan**
Any discovered bugs are in #680

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -530,3 +530,29 @@ source and target branches accordingly.
...
+Locust is a dev requirement so running it is straight forward with your development
+environment set up.
+
+1. Make sure Locust is installed by running
+   ```
+   locust --version
+   ```
+   If it is not installed, do step 1.3 in this README.
+
+1. To run scale test the webservice on integration run
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
1. To scale test the Azul web service on integration run
```

File: README.md [hunk 2]
```diff
@@ -530,3 +530,29 @@ source and target branches accordingly.
...
+   ```
+   locust -f scripts/locust/service.py
+   ```
+
+   If you want to test against a different stage use the `--host` option:
+   ```
+   locust -f scripts/locust/service.py --host https://service.dev.explore.data.humancellatlas.org
+   ```
+
+1. Navigate to `localhost:8090` in your browser to start a test run.
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
1. Navigate to `http://localhost:8090` in your browser to start a test run.
```

File: lambdas/service/app.py [hunk 3]
```diff
@@ -521,7 +521,7 @@ def get_order():
 @app.route('/repository/files/export', methods=['GET'], cors=True)
 def get_manifest():
     """
-    Creates and returns a manifest based on the filters pased on
+    Creates and returns a manifest based on the filters passed on
```
Comment (hannes-ucsc) [inline lambdas/service/app.py]: Separate commit.

Comment (jessebrennan) [reply lambdas/service/app.py]: done

File: README.md (hunk 2 above)
Comment (jessebrennan) [reply README.md]: :+1:

File: README.md (hunk 1 above)
Comment (jessebrennan) [reply README.md]: :+1:

File: README.md [hunk 4]
```diff
@@ -530,3 +530,29 @@ source and target branches accordingly.
    that matches the target branch
 
 7) Run `git push origin`
+
+## 7 Scale testing
+
+Scale testing can be done with [Locust](https://locust.io/).
+Locust is a dev requirement so running it is straight forward with your development
```
[the remark this answers is not in the document]
Comment (jessebrennan) [reply README.md]: done.

**Comment by hannes-ucsc** [comment]
Fixed the commit titles and force pushed. Just waiting for green status.

**Pull request merged**